### DISCUSSION TITLE
Select all, select windows hover

### DISCUSSION THREAD

**Discussion by venetikidischar** [sa-si-dev/virtual-select#229] [Q&A] [answered]
1) Is there any way to close the selection menu automatically if user selects the all option?
2) Is there a way to close the selection menu automatically if mouse is out of the menu?

**Comment by sa-si-dev** [comment, marked as answer]
@venetikidischar These features not available in the plugin.
But you can close the dropbox programmatically using [close](https://sa-si-dev.github.io/virtual-select/#/methods?id=close) method whenever you want.
